doc/cephadm/octopus_gsg.rst

```diff
@@ -0,0 +1,262 @@
+        INFO:cephadm:Deploying alertmanager service with default placement...
+        INFO:cephadm:Enabling the dashboard module...
+        INFO:cephadm:Waiting for the mgr to restart...
+        INFO:cephadm:Waiting for Mgr epoch 13...
+        INFO:cephadm:Generating a dashboard self-signed certificate...
+        INFO:cephadm:Creating initial admin user...
+        INFO:cephadm:Fetching dashboard port number...
+        INFO:cephadm:Ceph Dashboard is now available at:
+
+        URL: https://192-168-1-113:8443/
+        User: admin
+        Password: 1nuk3cgx5m
+
+        INFO:cephadm:You can access the Ceph CLI with:
+
+        sudo /usr/sbin/cephadm shell --fsid 501812ec-a426-11ea-b71d-48f17fce7093 -c /etc/ceph/ceph.conf -k /etc/ceph/ceph.client.admin.keyring
+
+        INFO:cephadm:Please consider enabling telemetry to help improve Ceph:
+
+        ceph telemetry on
+
+        For more information see:
+
+        https://docs.ceph.com/docs/master/mgr/telemetry/
+
+        INFO:cephadm:Bootstrap complete.
+
+   For more information on bootstrapping a Ceph cluster, see :ref:`cluster_bootstrap`.
+
+2. Enter the cephadm shell::
+
+        [root@192-168-1-113 zdover]# cephadm shell
+        INFO:cephadm:Inferring fsid c56ac3b8-a4cc-11ea-82a0-48f17fce7093
+        INFO:cephadm:Using recent ceph image docker.io/ceph/ceph:v15
+
+   .. note::
+            As of Octopus it is not necessary to install `cephadm-shell`. `cephadm-shell` is installed during the bootstrap procedure. If this doesn't mean anything to you, know that it used to be necessary at this point in the installation procedure to install a program called `cephadm-shell` so that you could interact with Ceph, but the Ceph release called "Octopus" automatically installs that program, making its explicit installation unnecessary.
+
+
+3. Add the Octopus-release repository by running the ``cephadm add-repo`` command::
+
+        [ceph: root@192-168-1-113 /]# cephadm add-repo --release octopus
+        INFO:root:Writing repo to /etc/yum.repos.d/ceph.repo...
+        INFO:cephadm:Enabling EPEL...
+        INFO:cephadm:Enabling supplementary copr repo ktdreyer/ceph-el8...
+        [ceph: root@192-168-1-113 /]# 
+
+   .. note::
+            The above command took some time to run (~5 minutes).
+
+
+4. Ensure that ssh is enabled on the node on which the bootstrap command was run::
+
+        [zdover@192-168-1-102 ~]$ history
+        1 history
+        [zdover@192-168-1-102 ~]$ ssh zdover@192.168.1.113
+        zdover@192.168.1.113's password: 
+        Activate the web console with: systemctl enable --now cockpit.socket
+
+5. Confirm that Ceph is installed on the first node::
+
+        Last login: Tue May 26 16:27:59 2020
+        [zdover@192-168-1-113 ~]$ ceph -v
+        ceph version 15.2.2 (0c857e985a29d90501a285f242ea9c008df49eb8) octopus (stable)
+
+6. Use ``ceph status`` to check that the incipient system is in place::
```

Comment:
"incipient" raises the barrier to entry for ESL. Perhaps consider, "developing" "building" etc - obviously with better phrasing. But it's WIP.